src/main/java/core/net/DownloadFilter.java

```diff
@@ -54,9 +54,7 @@ private void fillOldFixturesList(DefaultMutableTreeNode previousSeriesData) {
             int finalI = i;
             var season = seasons.stream().filter(f -> f.getSaison() == finalI).findFirst();
             var itemText = new StringBuilder(TranslationFacility.tr("Season")).append(" ").append(i);
-            if (season.isPresent()) {
-                itemText.append(" / ").append(season.get().getLigaName());
-            }
+            season.ifPresent(spielplan -> itemText.append(" / ").append(spielplan.getLigaName()));
```

Comment:
this change is okay, but please do not reintroduce german attribute names

Reply:
spielplan -> fixture ???

Reply:
Fixed.